4-4-3.py

```diff
@@ -0,0 +1,39 @@
+'''
+След матрицы
+
+Следом квадратной матрицы называется сумма элементов главной диагонали. Напишите программу, которая выводит след заданной квадратной матрицы.
+
+Формат входных данных
+На вход программе подаётся натуральное число nnn — количество строк и столбцов в матрице, затем элементы матрицы (целые числа) построчно через пробел.
+
+Формат выходных данных
+Программа должна вывести одно число — след заданной матрицы.
+Тестовые данные 🟢
+
+Sample Input 1:
+
+3
+1 2 3
+4 5 6
+7 8 9
+
+Sample Output 1:
+
+15
+'''
+
+def matrix_trace(m_size):
+    matrix = []
+    trace = []
+    for elem in range(m_size):
+        temp = [int(num) for num in input().split()]
```

Comment:
Try always to give variables meaningful names reflecting theirs content. In this case, it's better to name it something like`row_strs`. And try to do this exercise for the variable above, `elem`. And even check all the variables in this PR and apply this suggestion.

Reply:
you suggest will declare a variable a trace after a loop?